deps/exokit-bindings/gluegen/glbindinit.h

```diff
@@ -399,3 +399,10 @@
 	_gl->Set(v8::String::NewFromUtf8(isolate, "vertexAttrib4f"), v8::FunctionTemplate::New(isolate, __vertexAttrib4f));
 	_gl->Set(v8::String::NewFromUtf8(isolate, "vertexAttribPointer"), v8::FunctionTemplate::New(isolate, __vertexAttribPointer));
 	_gl->Set(v8::String::NewFromUtf8(isolate, "viewport"), v8::FunctionTemplate::New(isolate, __viewport));
+	_gl->Set(v8::String::NewFromUtf8(isolate, "createQuery"), v8::FunctionTemplate::New(isolate, __createQuery));
```

Comment:
I don't think we need this, since this file is not currently used in the build (used to generate mobile bindings, which is a separate issue).